### ISSUE TITLE
Some characters in filenames break the video uploader

### ISSUE THREAD

**Issue by glubsy** [Chocobozzz/PeerTube#4984]
**Current behavior**

When attempting to upload a video, some characters in the filename appear to break the uploader.
Note that these characters don't cause any issue: `;` `,` `'`.

**Steps to reproduce:**

This may only affect the Peertube instance on which I have tested this (namely ploud.fr)

1. Have a filename with any one of the following characters in it: `|` `?` `:` `\` `"` `*` `>` `<` possibly also `/`
2. Go to "Publish" a video, click "Select the file to upload", 
3. Select the file with the file picker modal window in the video uploader and click OK / Open
4. A popup message will say `The upload failed, [object Object]` and the progress bar will now display `[object Object]` with a red background

**Describe the expected behavior**

The offending characters should be preserved as is, or stripped, or converted to something like `_` maybe.

**Additional information**

* PeerTube instance:
  * URL: https://video.ploud.fr/videos/upload
  * version: 4.1.0
  * NodeJS version: ? not publicly disclosed
  * Ffmpeg version: ? not publicly disclosed

* Chromium, Version 101.0.4951.64 (Official Build) Arch Linux (64-bit)
* Link to browser [console log](https://gist.github.com/glubsy/49916e59c08a2353676b8d2020a2c256) (note that there are multiple instances of the errors as I was testing uploading with different "forbidden" characters)

**Comment by kukhariev** [comment]
same fix as #4979

It is also possible to fix this unclear error ` [object Object]`:

https://github.com/Chocobozzz/PeerTube/blob/f15f20a5be725a324f0ff7f9abda7dba3da4c4c6/client/src/app/%2Bvideos/%2Bvideo-edit/video-add-components/video-upload.component.ts#L139

```ts
const error = state.response?.error?.message || state.response?.error || 'Unknown error' 
```

**Comment by Chocobozzz** [comment]
Thanks @kukhariev 

I added your fix in https://github.com/Chocobozzz/PeerTube/commit/f4120aea7a4b4785cc59b690b9187621c84126af
